how-to/customize-workspace/public/windows/version/version.js

```diff
@@ -70,35 +76,47 @@ async function init() {
 		const minFail = customData.minFail ?? [];
 		const maxFail = customData.maxFail ?? [];
 
-		const keys = Object.keys(versionInfo);
-
-		for (let i = 0; i < keys.length; i++) {
-			const currentKey = keys[i];
-			const minVersionMet = minVersion[currentKey] === undefined || !minFail.includes(currentKey);
-			const maxVersionMet = maxVersion[currentKey] === undefined || !maxFail.includes(currentKey);
-			addVersion(
-				currentKey,
-				versionInfo[currentKey],
-				minVersion[currentKey],
-				minVersionMet,
-				maxVersion[currentKey],
-				maxVersionMet
-			);
+		if (minFail.includes('app')) {
+			const btnRestart = document.querySelector('#restart');
+			const currentVersion = document.querySelector('#current-version');
+			const newVersion = document.querySelector('#new-version');
+			btnRestart.addEventListener('click', async () => {
+				await restartPlatform();
+			});
+			currentVersion.textContent = versionInfo['app'];
+			newVersion.textContent = minVersion['app'];
+		} else {
+			const keys = Object.keys(versionInfo);
+
+			for (let i = 0; i < keys.length; i++) {
+				const currentKey = keys[i];
+				const minVersionMet = minVersion[currentKey] === undefined || !minFail.includes(currentKey);
+				const maxVersionMet = maxVersion[currentKey] === undefined || !maxFail.includes(currentKey);
+				addVersion(
```

Comment:
Version display fallback to "unknown" if version not available

Reply:
think I did what you were after.